airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```
@@ -173,23 +174,23 @@
                         "default": "zstandard"
                       },
                       "compression_level": {
-                        "title": "Compression level",
+                        "title": "Compression Level",
                         "description": "Negative levels are 'fast' modes akin to lz4 or snappy, levels above 9 are generally for archival purposes, and levels above 18 use a lot of memory.",
                         "type": "integer",
                         "default": 3,
                         "minimum": -5,
                         "maximum": 22
                       },
                       "include_checksum": {
-                        "title": "Include checksum",
+                        "title": "Include Checksum",
                         "description": "If true, include a checksum with each data block.",
                         "type": "boolean",
                         "default": false
                       }
                     }
                   },
                   {
-                    "title": "snappy",
+                    "title": "Snappy",
```

```suggestion
                    "title": "snappy",
```
These types are always in lower case, I don't think it is necessary to put them in upper case.